### ISSUE TITLE
[Blockly Developer Tools] developer tools on demo website dead

### ISSUE THREAD

**Issue by ppodds** [google/blockly#6496]
**Describe the bug**

The tools seem dead, but it worked in few days ago.

**To Reproduce**

Open website: [https://blockly-demo.appspot.com/static/demos/blockfactory/index.html](https://blockly-demo.appspot.com/static/demos/blockfactory/index.html)

**Expected behavior**

Developer tools should work like documents.

**Screenshots**

![image](https://user-images.githubusercontent.com/11408727/194347457-406f0cbb-7d83-4c04-83d2-b5458597bf2f.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Chrome / Edge (be tested)

**Stack Traces**

```
Uncaught TypeError: Blockly.utils.genUid is not a function
    at new ListElement (wfactory_model.js:480:27)
    at standard_categories.js:28:5
wfactory_model.js:480
Uncaught TypeError: Blockly.utils.genUid is not a function
    at new ListElement (wfactory_model.js:480:27)
    at new WorkspaceFactoryModel (wfactory_model.js:26:17)
    at new WorkspaceFactoryController (wfactory_controller.js:59:16)
    at new AppController (app_controller.js:25:37)
    at init (index.html:40:24)
```

**Additional context**

**Comment by its-pablo** [comment]
This is being fixed with #6493

**Comment by maribethb** [comment]
This is now fixed and is updated on app engine.
